trio/_tests/test_deprecate.py

```diff
@@ -11,23 +14,35 @@
 )
 from . import module_with_deprecations
 
+if TYPE_CHECKING:
+    from typing_extensions import TypeAlias
+
+
+T = TypeVar("T")
+
 
 @pytest.fixture
-def recwarn_always(recwarn):
+def recwarn_always(recwarn: T) -> T:
```

Comment:
Actually, `recwarn` is [pytest.WarningsRecorder](https://docs.pytest.org/en/stable/reference/reference.html#pytest.recwarn.recwarn), so we can drop the typevar and alias.

Reply:
Changed in c06e788